### DISCUSSION TITLE
农历倒数日

### DISCUSSION THREAD

**Discussion by s1qhyr** [WatsonYao/nian_app#64] [Q&A] [answered]
- 使用场景：用以记录亲人朋友的农历生日，因此需要能按农历日期记录并按农历年为单位重复

- 不过搜索发现#23 中提到“删除农历相关的计算和显示”，不知道是否还会考虑增加农历相关功能

- PS：顺便感谢开发出这么好的产品

**Comment by WatsonYao** [comment, marked as answer]
感谢反馈，农历不再支持，每年花点时间给亲朋好友添加一次生日也蛮有仪式感的。
